.travis.yml

```diff
@@ -1,94 +1,98 @@
-# This file has been generated -- see https://github.com/hvr/multi-ghc-travis
+# This Travis job script has been generated by a script via
```

Comment:
Is `cabal build` officially deprecated? Does it produce warnings on the latest stable version of `cabal`?